test/cypress/e2e/search.spec.js

```diff
@@ -64,9 +48,9 @@ describe('Search', () => {
   it('Search for Lisk ID using suggestions, signed in', () => {
     cy.autologin(accounts.genesis.passphrase, networks.devnet.node);
     cy.visit(urls.dashboard);
-    cy.get(ss.searchInput).click().type(`${accounts.delegate.address}`);
-    cy.get(ss.idResults).eq(0).click();
```

Comment:
You removed the essence of the test. Read the name of the test. What are you testing now?